cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -154,7 +154,7 @@ func getKubeConfigSpecs(cfg *kubeadmapi.InitConfiguration) (map[string]*kubeConf
 		kubeadmconstants.KubeletKubeConfigFileName: {
 			CACert:     caCert,
 			APIServer:  controlPlaneEndpoint,
-			ClientName: fmt.Sprintf("%s%s", kubeadmconstants.NodesUserPrefix, cfg.NodeRegistration.Name),
+			ClientName: fmt.Sprintf("%s %s", kubeadmconstants.NodesUserPrefix, cfg.NodeRegistration.Name),
```

Comment:
can you please explain your change here?
how is this a bug?

@kubernetes/sig-cluster-lifecycle-pr-reviews

Reply:
Maybe I was wrong , I think there should be a space between the two parameters.

Reply:
@neolit123  Shall I change it back?

Reply:
This should not be changed. See https://kubernetes.io/docs/reference/access-authn-authz/node/